src/MetricsMsg.chpl

```diff
@@ -219,7 +228,7 @@ module MetricsMsg {
             return value;
         }
         
-        proc add(metric: string, measurement) throws {
+        override proc add(metric: string, measurement: real) throws {
```

Comment:
Just noting if we did update the name of the new add function, this could remain as it was or also rename to that.